### ISSUE TITLE
Error in applying plugin from a shared script

### ISSUE THREAD

**Issue by rzabini** [tschulte/gradle-semantic-release-plugin#8]
Hi, 
I was trying to apply the plugin from a shared script, which also contains the buildscript classpath. 
In this script I also apply the `versionStrategy RebuildVersionStrategy.INSTANCE` to the `release` task configuration.

I found [here](https://discuss.gradle.org/t/how-do-i-apply-a-plugin-to-a-project-from-a-shared-applied-gradle-file/7508) that in this case:

> ... you have to use the plugin's fully qualified class name, rather than its ID

So I used this:

```
project.apply plugin: de.gliderpilot.gradle.semanticrelease.SemanticReleasePlugin
```

But the got an error on the application of `org.ajoberstar.grgit.*` (two) plugins, because they are applied by name in the `apply` method of class `SemanticReleasePlugin`.

I was able to make it work by using the fully qualified class name and rebuilding the semantic-release plugin:

```
plugins.apply org.ajoberstar.gradle.git.release.base.BaseReleasePlugin
plugins.apply org.ajoberstar.gradle.git.base.GrgitPlugin
```

What do you think?
Thank you.

**Comment by tschulte** [comment]
Yes, that would be OK. @rzabini Do you want to create a Pull-Request?

**Comment by tschulte** [comment]
closed with d0ae51cac3858016634b6b82aa91bdd033886950
